### ISSUE TITLE
Date time dont work in my fullcalendar

### ISSUE THREAD

**Issue by azzeddine05** [maddhatter/laravel-fullcalendar#124]
When i fetch my data to fullcalendar the data dispalyed correctly but when i change format to datetime dont work, i want to disoplay my data by hours and minutes ly default view : agendaweek please any help
this is my code : 

                   $events = Agenda::where('code_commercial', $codeCommercial)->get();
                    $event_list = [];
                    foreach ($events as $event) {
                         $event_list[] = Calendar::event(
                        $event->description . " " . $event->cp,
                        true,
                       // date_format(date_create($event->date_debut), "Y-m-d H:i:s"),
                      // date_format(date_create($event->date_fin), "Y-m-d H:i:s")
                      // // new \DateTime($event->date_debut),
                     // new \DateTime($event->date_fin),
                    Carbon::createFromFormat('Y-m-d H:i:s', $event->date_debut),
                    Carbon::createFromFormat('Y-m-d H:i:s', $event->date_fin),
                    $event->id
                );
        }

**Comment by websiteproduct** [comment]
@azzeddine05 
```
$event_list[] = Calendar::event(
$event->description . " " . $event->cp,
true,
```

true - it's mean a full-day event. Use false instead of true.

```
$event_list[] = Calendar::event(
$event->description . " " . $event->cp,
false,
```

**Comment by azzeddine05** [comment]
@websiteproduct  yes finaly it work just change true to false  thanks thanks @websiteproduct
